rstudio/rstudio
# Remove 'Classic' theme and supporting resources

### Intent

This change removes the RStudio Classic theme, with the goal of addressing (in the most brutal way) a couple of issues with this theme while reducing the test matrix and code complexity.

Addresses https://github.com/rstudio/rstudio/issues/9738.

### Approach

If this change looks dramatic, it is: the two "modern" themes are, as it turns out, just wallpaper over the Classic theme. Many decorative images, elements, and styles that make up the Classic theme start out visible but are then hidden, removed, or overridden by the modern themes. 

As part of the goal of this change is to reduce surface area and improve code quality, I've gone through and removed all of the decorative images and, where appropriate, entire HTML elements that were only used to support Classic. CSS rules only active in Classic have been removed and recombined. This made the change quite a lot riskier but also leaves us with very little (hopefully no) vestigial code. 

For CSS files (the bulk of this change), the procedure is to remove the `.rstudio-themes-flat` selector and then push any rules it contained as new defaults, working backwards and removing images, then elements that are no longer referenced.

### Automated Tests

No new functionality is introduced by this PR, so nothing new to test. 

### QA Notes

The good news is that this change is almost purely visual; the bad news is that anything visual could be affected. I would recommend a quick sanity pass over all windows and dialogs, spot checking various combinations and looking for color or alignment issues:

- Dark vs. Light
- Desktop vs. Server
- Modern vs. Sky

While undergoing this work I noticed a number of visual nits, some of which I corrected ... and some of which I let be. Before filing a bug, compare to a copy before the change to so you can note that it's a regression (but file the bug either way!). 

If a user previously had the Classic theme enabled, they should see "Modern" show up instead. 

### Checklist
